### ISSUE TITLE
The problem of the logits'shape in the train_epoch function in the dgl_main.py

### ISSUE THREAD

**Issue by fantasy-zc** [bdy9527/Specformer#7]
Hi, i am very excited for your work, so i am running your codes. there is a problem for me in the train_epoch function in the dgl_main.py.
Actually, in the train_epoch, there is a parameter named as "logits", the computation of logits as follow
`logits = model(e, u, g, length)`
so logits'type is tuple, so there will be a error such as "'tuple' object has no attribute 'to'".
Now i solve this problem by chatgpt
` #chatgpt
        if isinstance(logits, tuple):
            logits = logits[0]
#chatgpt
        # Adjust logits and y shapes if necessary
        logits = logits.squeeze(dim=-1)  # 去除多余的维度`
By adding these codes, the program can run.
However, logits is a tuple, the code`logits = logits[0]` just gains the first element of logits, is this ture?
I will be happy for your reply!
Best wishes!

**Comment by Lukangkang123** [comment]
I think so!
